libcxx/test/std/localization/locale.categories/category.numeric/locale.num.get/facet.num.get.members/get_double.pass.cpp

```diff
@@ -176,9 +182,9 @@ int main(int, char**)
             f.get(cpp17_input_iterator<const char*>(str),
                   cpp17_input_iterator<const char*>(str+sizeof(str)),
                   ios, err, v);
-        assert(base(iter) == str+sizeof(str)-1);
-        assert(err == ios.goodbit);
-        assert(std::isnan(v));
+        assert(base(iter) == str);
+        assert(err == ios.failbit);
+        assert(v == 0.0);
     }
```

Comment:
Can you also test with "p00", "+p00", "-p00" and similar for "P00", "e00", and "E00"?

Reply:
Oh, these stuffs are now completely accumulated, which seems to be pre-existing. Presumably the leading 'p' or 'e' shouldn't be accumulated ([[facet.num.get.virtuals]/3](https://eel.is/c++draft/facet.num.get.virtuals#3.2.sentence-5)).

Reply:
I'll try to fix the checking for the leading sign and digit. But there'rs still non-trivial issues, which perhaps should be fixed in another PR. Reported #87969.

Reply:
> Can you also test with "p00", "+p00", "-p00" and similar for "P00", "e00", and "E00"?

These cases are now fixed.